templates/c2s-lb/config.yml

```diff
@@ -1,6 +1,6 @@
 name: C2S - Load Balancer
 description: |
   Connecting to Services - Load Balancer config for staging/production
-version: v1
-category: nhsuk beta
+version: 1.0.0
+category: nhsuk beta load-balancer
 projectURL: https://github.com/nhsuk/nhsuk-rancher-templates
```

Comment:
Based on this v1 does relate to something as it's not maintained by us: https://rancher.nhschoices.net/env/1a7/catalog/NHSuk:traefik?catalogId=NHSuk